README.md

```diff
@@ -1,5 +1,8 @@
 # BlueSky : Web dev in Eclipse IDE
 
+[![Eclipse License](http://img.shields.io/badge/license-Eclipse-brightgreen.svg)](https://github.com/mickaelistria/eclipse-bluesky/blob/master/LICENSE)
+[![Build Status](https://secure.travis-ci.org/mickaelistria/eclipse-bluesky.png)](http://travis-ci.org/mickaelistria/eclipse-bluesky)
```

Comment:
I would rather see that badge at the bottom of the page, with the `Support` area. as the top of the page is targeting end-users, not contributors.